### PR TITLE
Drop JSON2 dependancy

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

**Review by dehann — APPROVED** [review]
Happy for you to merge (understood from the call that this change works for all tests downstream IIF, RoME, etc.)

**Comment by dehann** [comment]
oh, and awesome that you were able to get away from JSON2, i tried but was not able to :-P

File: src/services/Serialization.jl [hunk 1]
```diff
@@ -248,24 +248,11 @@ function decodePackedType(dfg::AbstractDFG, varOrder::AbstractVector{Symbol}, ::
...
-    
-    # TODO use kwdef constructors instead,
-    # @info "user factor" fncData["fnc"]
-    restring = JSON3.write(fncData["fnc"])
-    # @show restring
-    # packT = JSON3.read(restring, packtype)
-    packT = packtype(;JSON2.read(restring)...)
+    # Read string as JSON object to use as kwargs
+    fncData = JSON3.read(data)
+    packT = packtype(;fncData.fnc...)
```
Comment (dehann) [inline src/services/Serialization.jl:255]: Uhmm, this did not work for me when I tried this.  Maybe you fixed the issue somewhere else

Comment (Affie) [reply src/services/Serialization.jl:255]: I made a change in iif also

**Comment by Affie** [comment]
IIF was fine RoME still has failures. I’ll merge and then fix downstream.

**Pull request merged**